contrib/stack/topsStack/plotBursts.py

```diff
@@ -138,11 +138,18 @@ def main(iargs=None):
                 lateNear = burst.orbit.rdr2geo(t1,r0)
 
                 ring = ogr.Geometry(ogr.wkbLinearRing)
-                ring.AddPoint(earlyNear[1], earlyNear[0])
-                ring.AddPoint(earlyFar[1], earlyFar[0])
-                ring.AddPoint(lateFar[1], lateFar[0])
-                ring.AddPoint(lateNear[1], lateNear[0])
-                ring.AddPoint(earlyNear[1], earlyNear[0])
+                if int(gdal.__version__[0]) >= 3:
```

Comment:
Looking at the first character of the version string seems potentially brittle. Maybe we can use distutils for a more robust version comparison?
```suggestion
                from distutils.version import StrictVersion
                if StrictVersion(gdal.__version__) >= StrictVersion("3.0")
```